tests/test_plotting.py

```diff
@@ -396,62 +375,55 @@ def test_section_J(DSHAPE_current):
 
 
 @pytest.mark.unit
-@pytest.mark.solve
 @pytest.mark.mpl_image_compare(remove_text=True, tolerance=24)
-def test_section_Z(DSHAPE_current):
+def test_section_Z():
```

Comment:
isn't this basically redundant with the R plot, since no new code should be exercised if plotting R or Z, just a different quantiity is computed?